<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorHandlerEditor</class>
 <widget class="QWidget" name="ColorHandlerEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>496</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Stroke</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelPenWidth">
        <property name="text">
         <string>Width:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="sbWidth">
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPenColor">
        <property name="text">
         <string>Color:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="shared::ColorSelectorButton" name="btnColorStroke">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxPenStyle"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelPenStyle">
        <property name="text">
         <string>Style:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Fill</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelFillType">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbFillType"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelBrushColor0">
        <property name="text">
         <string>Color:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="shared::ColorSelectorButton" name="btnColor">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelBrushColor1">
        <property name="text">
         <string>Stop color:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="shared::ColorSelectorButton" name="btnColorStop">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Text</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="labelFontFamily">
        <property name="text">
         <string>Font:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFontComboBox" name="fontComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelFontPointSize">
        <property name="text">
         <string>Point size:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxFontSize"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelTextColor">
        <property name="text">
         <string>Color:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelTextAlignment">
        <property name="text">
         <string>Alignment:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBoxTextAlignment"/>
      </item>
      <item row="3" column="1">
       <widget class="shared::ColorSelectorButton" name="btnTextColor">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelFontWeight">
        <property name="text">
         <string>Weight:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxFontWeight"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>shared::ColorSelectorButton</class>
   <extends>QToolButton</extends>
   <header>colors/colorselectorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
